tests/integration_tests/tests/agentless_tests/test_rest_service_list_filters.py

```diff
@@ -200,15 +203,15 @@ def test_plugins_list_with_filters(self):
 
     def test_plugins_list_no_filters(self):
         test_utils.upload_mock_plugin(
-                self.client,
-                TEST_PACKAGE_NAME,
-                TEST_PACKAGE_VERSION)
+            self.client,
+            TEST_PACKAGE_NAME,
+            TEST_PACKAGE_VERSION)
         test_utils.upload_mock_plugin(
-                self.client,
-                TEST_PACKAGE_NAME,
-                OLD_TEST_PACKAGE_VERSION)
-        response = self.client.plugins.list()
+            self.client,
+            TEST_PACKAGE_NAME,
+            OLD_TEST_PACKAGE_VERSION)
+        plugins = [(p.package_name, p.package_version)
+                   for p in self.client.plugins.list()]
 
-        self.assertEqual(len(response), 2, 'expecting 2 plugin results, '
```

Comment:
plugins are a session-level thing, there might well be more than 2 depending on the tests selected